frontend/src/surveys/utils/http.js

```diff
@@ -0,0 +1,22 @@
+import { ServerUrl } from '../../server.config';
+import { Survey } from '../../models/survey';
+
+const axios = require('axios');
+
+
+class HttpUtility {
+    static async GetSurveys() {
```

Comment:
this file should not make calls to server. You should create separate axios instance here that instance. 
https://github.com/axios/axios#creating-an-instance

You can also configure global error trapping here 
https://github.com/axios/axios#interceptors